src/lib/dev_mockData/cardPubKeys.ts

```diff
@@ -0,0 +1,37 @@
+export type CardPubKey = {
+  pubKeyWeierstrass: string;
+  pubKeyEdwards: string;
+  index: number;
+  name: string;
+  imagePath: string;
+};
+
+export const cardPubKeys: Record<string, CardPubKey> = {
```

Comment:
Should we replace these with the pubkeys I just sent?

Reply:
Yuppers